openlane/steps/magic.py

```diff
@@ -449,3 +448,58 @@ def run(self, state_in: State, **kwargs) -> Tuple[ViewsUpdate, MetricsUpdate]:
         )
 
         return views_updates, metrics_updates
+
+
+@Step.factory.register()
+class OpenGUI(MagicStep):
+    """
+    Opens the DEF view in the Magic GUI.
+    """
+
+    id = "Magic.OpenGUI"
+    name = "Open In GUI"
+
+    inputs = [DesignFormat.DEF]
+    outputs = []
+
+    config_vars = MagicStep.config_vars + [
+        Variable(
+            "MAGIC_PRIORITIZE_GDS",
```

Comment:
Taken out of context of the step being ran, the name of the variable is confusing. I am not sure if that's something we need to consider tho. What do you think?

Reply:
It's consistent with KLayout's `OpenInGUI` step but it's not consistent with Magic's other steps which are like `MAGIC_GDS_`

You are right that this may cause some confusion. Perhaps both variables should be renamed.